src/index.d.ts

```diff
@@ -4088,7 +4089,7 @@ declare namespace admin.messaging {
      * For example, this might be useful if you're using just one notification to
      * represent multiple new messages but you want the count here to represent
      * the number of total new messages. If zero or unspecified, the systems
-     * that support badging uses it's default behavior, which is to increment
+     * that support badging use it's default behavior, which is to increment
```

Comment:
Sorry, missed this: it's >> its.

Counterintuitive but true, it's = "it is," not possession.

Reply:
Thanks, @egilmorez! This is an embarrassing one!
I updated the docs here to match with `https://firebase.google.com/docs/reference/fcm/rest/v1/projects.messages#androidnotification`. Let me know what you think. Thanks!